Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"ProjectModuleManager\">\n    <modules>\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/seb42_main_027.iml\" filepath=\"$PROJECT_DIR$/.idea/seb42_main_027.iml\" />\n      <module fileurl=\"file://$PROJECT_DIR$/.idea/modules/server.main.iml\" filepath=\"$PROJECT_DIR$/.idea/modules/server.main.iml\" />\n    </modules>\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
--- a/.idea/modules.xml	(revision 2fceb1118deaa110fd84dc58ec0c3efa9ed50ec0)
+++ b/.idea/modules.xml	(date 1678948702194)
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/.idea/seb42_main_027.iml" filepath="$PROJECT_DIR$/.idea/seb42_main_027.iml" />
       <module fileurl="file://$PROJECT_DIR$/.idea/modules/server.main.iml" filepath="$PROJECT_DIR$/.idea/modules/server.main.iml" />
+      <module fileurl="file://$PROJECT_DIR$/.idea/modules/server.test.iml" filepath="$PROJECT_DIR$/.idea/modules/server.test.iml" />
     </modules>
   </component>
 </project>
\ No newline at end of file
